modules/apps/object/object-api/src/main/java/com/liferay/object/field/filter/parser/OneToManyObjectFieldFilterStrategy.java

```diff
@@ -143,7 +143,7 @@ public List<Map<String, Object>> getItemsValues() throws PortalException {
 		}
 
 		for (int i = 0; i < jsonArray.length(); i++) {
-			ObjectEntry objectEntry = _objectEntryLocalService.getObjectEntry(
+			ObjectEntry objectEntry = _objectEntryLocalService.fetchObjectEntry(
```

Comment:
Isn't there a chance to throw a null pointer? 
line below:
`itemsValues.add(
	HashMapBuilder.<String, Object>put(
			"label", objectEntry.getTitleValue()
	).put(
			"value", objectEntry.getObjectEntryId()
	).build());
`

Reply:
Do you think it would be better to check if the object entry differs from null?

Reply:
> Do you think it would be better to check if the object entry differs from null?

Done!